.github/workflows/build_and_test.yml

```diff
@@ -108,7 +91,47 @@ jobs:
             --google_credentials=${{ env.GOOGLE_APPLICATION_CREDENTIALS }}" \
             >> $GITHUB_ENV
 
-      - name: Build
+      # Use separate build steps to use less memory.
+      - name: Build eventuals
+        run: |
+          ${{ matrix.bazel-command }} \
+            ${{ matrix.output-user_root }} \
+            build \
+            ${{ matrix.bazel-config }} \
+            ${{ env.BAZEL_REMOTE_CACHE }} \
+            --verbose_failures \
+            -c dbg \
+            --strip="never" \
+            test:eventuals
+
+      - name: Build eventuals grpc
+        if: runner.os != 'Windows'
+        run: |
+          ${{ matrix.bazel-command }} \
+            ${{ matrix.output-user_root }} \
+            build \
+            ${{ matrix.bazel-config }} \
+            ${{ env.BAZEL_REMOTE_CACHE }} \
+            --verbose_failures \
+            -c dbg \
+            --strip="never" \
+            test/grpc:grpc
+
+      - name: Build route-guide
+        if: runner.os != 'Windows'
+        run: |
+          ${{ matrix.bazel-command }} \
+            ${{ matrix.output-user_root }} \
+            build \
+            ${{ matrix.bazel-config }} \
+            ${{ env.BAZEL_REMOTE_CACHE }} \
+            --verbose_failures \
+            -c dbg \
+            --strip="never" \
+            test/grpc/route_guide:route-guide-test
```

Comment:
Did you say this splitting into multiple steps didn't have any effect on the memory usage? If so, can we revert this part of the change?

Reply:
I’m doing tests now, recently found some additional hazel flags that might help, will revert if it would be not necessary, thank you!